### DISCUSSION TITLE
PDF reader

### DISCUSSION THREAD

**Discussion by heyhippari** [jellyfin/jellyfin-vue#149] [Features]

**Comment by AlfHou** [comment]
Is this avaliable in jellyfin-web? Can't seem to get it to work

**Comment by camc314** [comment]
It was only merged ~1 week ago so won't be working in 10.6.4

I'd you try master, it should work

https://github.com/jellyfin/jellyfin-web/pull/1887

**Comment by AlfHou** [comment]
I see! I'm using my "prod" server for hosting the backend, I don't thing my 2012 Macbook can handle hosting both the server and the frontend 😝
